src/games/strategy/debug/ClientLogger.java

```diff
@@ -38,4 +40,14 @@ public static void logError(final String msg, final Throwable e) {
     logError(msg);
     logError(e);
   }
+
+  public static void logError(LocalSystemChecker systemCheck) {
```

Comment:
Do you think we'll use this method in more places in the future? If not then it might be better to move this logic into LocalSystemChecker and have it return a string message and a set of exceptions. So this method would become logError(String msg, Set<Exception>) as I can see that being more reusable.